zb-db/src/main/java/io/zeebe/db/impl/rocksdb/ZeebeRocksDb.java

```diff
@@ -164,8 +164,8 @@ protected void put(long columnFamilyHandle, DbKey key, DbValue value) {
             value.getLength(),
             columnFamilyHandle);
       }
-    } catch (Exception ex) {
-      throw new RuntimeException(ex);
+    } catch (Exception e) {
+      throw new RuntimeException("Unexpected error occurred trying to write RocksDB entry", e);
```

Comment:
This way it is again other way round. Actual then expected.

Reply:
I see the wording "Unexpected error occurred trying to ..." as a way of implying the expectation (i.e. no error should have occurred), since the error is unexpected and the real information is in the cause. If that's not obvious enough though we could write "Expected to write a RocksDB entry, but an error occurred", but that felt contrived to me. I'm fine with writing it, I just thought it would look weird to others as well.